board/app/controllers/comment_controller.php

```diff
@@ -0,0 +1,73 @@
+<?php
+class CommentController extends AppController {
+
+const MAX_COMMENT_PER_PAGE = 5;
+
+    public function write() 
+    {
+        $thread_id = Param::get('thread_id');
+
+        $thread = Thread::get($thread_id);
+        $comment = new Comment();
+        $current_page = Param::get('page_next', 'write');
+
+        switch($current_page) { 
+            case 'write':
+            break;
+
+        case 'write_end':                
+            $comment->body = Param::get('body');
+            $comment->username = Param::get('username');
```

Comment:
for L19-20, you may make use of contruct method of the `Comment` model, please do the same to other parts of your code :bow:
```php
$data = array(
    'body' => Param::get(...)
    ...
);

$comment = new Comment($data);
```